pkg/sessions/cookie/session_store_test.go

```diff
@@ -147,16 +147,25 @@ func Test_splitCookie_joinCookies(t *testing.T) {
 	}
 	value := strings.Repeat(string(v), 1000)
 
-	for _, nameSize := range []int{1, 10, 50, 100, 200, 254} {
-		t.Run(fmt.Sprintf("%d length cookie name", nameSize), func(t *testing.T) {
-			cookie := &http.Cookie{
-				Name:  strings.Repeat("n", nameSize),
-				Value: value,
-			}
-			splitCookies := splitCookie(cookie)
-			joinedCookie, err := joinCookies(splitCookies)
-			assert.NoError(t, err)
-			assert.Equal(t, *cookie, *joinedCookie)
-		})
+	nameSizes := []int{1, 10, 50, 100, 200, 254}
```

Comment:
I think we should split these two test scenarios and not try to splice them together.

Makes it cleaner for the size test ensuring nothing breaks at the 256 max & your new trailing suffix test without them impacting eachother potentially.